src/PollinationSDK/Api/RegistriesApi.cs

```diff
@@ -62,8 +62,9 @@ public interface IRegistriesApi : IApiAccessor
         /// <param name="type"></param>
         /// <param name="name"></param>
         /// <param name="digest"></param>
-        /// <returns>AnyOfRecipePlugin</returns>
-        AnyOf<Recipe,Plugin> GetPackageJson (string owner, string type, string name, string digest);
+        /// <param name="baked"> (optional, default to false)</param>
+        /// <returns>AnyOfRecipePluginBakedRecipe</returns>
+        AnyOfRecipePluginBakedRecipe GetPackageJson (string owner, string type, string name, string digest, bool? baked = default);
```

Comment:
Failed to replace `AnyOfRecipePluginBakedRecipe `to   `AnyOf<Recipe,Plugin,BakedRecipe>`